src/components/Card.js

```diff
@@ -15,13 +27,25 @@ export default class Card {
   }
 
   _handleLike = () => {
-    this._likeButton.classList.toggle("card__like-button_active");
+    if (this._likeButton.classList.contains("card__like-button_active")) {
+      this._unlikeHandler(this._cardId);
+      this._likeButton.classList.remove("card__like-button_active");
```

Comment:
`this._likeButton.classList.remove("card__like-button_active");`

You need to **toggle likes** only in block `then` after a successful response from the server because if there is an error the user will not know about that and will see the toggled heart color.